RELEASE.md

```diff
@@ -26,6 +26,11 @@ All of the below changes apply to kmapper's D3 html visualization.
     - Multiple node color functions can be specified, and toggled between interactively in the display.
 - The toolbar display now uses css flexbox, which avoids overlap-problems on smaller viewports.
 
+#### Kmapper
+
+- change several visualize-related functions to be private
```

Comment:
Could you add version number here like the ones below?

Reply:
That's a l4 heading, a subsection of the L3 2.0.0

Reply:
We could just remove both of the L4's if you prefer -- the `#### Visualization` and `#### Kmapper`

Reply:
ahh yeah sorry I missed that there were multiple parts of the release notes! my bad.